scripts/templates/create-page/EmptyPageBase.mustache

```diff
@@ -0,0 +1,95 @@
+const baseUrl =
+  'https://onedrive.visualstudio.com/Design/_git/ui-fabric-website-internal?path=/apps/fabric-website-internal/src/pages/{{pagePath}}/';
+
+export const {{pageName}}PageBase: React.StatelessComponent<I{{pageName}}PageProps> = props => {
+  return (
+    <PlatformContext.Consumer>
+      {(platform: Platforms) => {
+        const { theme, styles, className } = props;
+
+        const classNames = getClassNames(styles, { theme, className });
+
+        return (
+          <Page
+            // Pass all the props to the Page component.
+            {...props}
+            // Set default page title
+            title="{{pageName}}"
+            // Use the platform specific props from the doc.ts file.
+            {...{{pageName}}PageProps[platform]}
+            // Use the getSubTitle helper function to get the page header subtitle from the active platform.
+            subTitle={getSubTitle(platform)}
+            // You can define custom sections using the `otherSections` prop. Here it is using a method that takes the platform as an argument to return the correct array of section props.
+            otherSections={_otherSections(platform)}
+            // You can pass a custom className to the page wrapper if needed.
+            className={classNames.root}
+
+            // You can hide the side rail by setting `showSideRail` to false.
+            // showSideRail={false}
+          />
+        );
+      }}
+    </PlatformContext.Consumer>
+  );
+};
+
+// Method that returns array of sections. Renders in the order defined.
+function _otherSections(platform: Platforms): IPageSectionProps[] {
+  // Use a switch statement to define the sections for each platform.
+  switch (platform) {
+    case 'web':
+      return [];
+
+    case 'ios':
+      return [];
+
+    case 'android':
+      return [];
+
+    default:
+      return [
+        // Custom section with content from markdown file.
+        {
+          // Optionally define the name of the section.
+          sectionName: 'Custom markdown section',
+
+          // If the content of the section is markdown, define the editUrl to enable the edit button as the file's hosted location.
+          editUrl: baseUrl + 'docs/default/{{pageName}}Custom.md',
+
+          // Define the content of the page using the `Markdown` component.
+          // Use raw-loader to point the markdown file you want to use for the section.
+          content: require('!raw-loader!@uifabric/fabric-website-internal/src/pages/{{pagePath}}/{{pageName}}Page/docs/default/{{pageName}}Custom.md') as string
+        },
+
+        // Custom section with JSX content.
+        {
+          // Optionally define the name of the section.
+          sectionName: 'Custom JSX section',
+
+          // Since there won't be a markdown file to edit online, don't define the editUrl.
+          // editUrl: undefined,
+
+          // Define the content using any React node you want.
+          content: (
```

Comment:
This can also be a string, which will be converted to Markdown (I added that a couple weeks ago)

Reply:
Added that to the comment.